### PR TITLE
Improve client initialization

### PR DESCRIPTION
**Pull request by MukjepScarlet**
By using multithreading, the client initialization time was reduced from 30 seconds to 20 seconds on my computer. (From starting the Java process to entering the main menu)

*Need to check if there are any hidden bugs, although I didn't find any problems after playing for few hours*

### CODE REVIEW

**Review by 1zun4 — APPROVED** [review]

File: src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/protocol/rest/v1/game/ServerListFunctions.kt [hunk 1]
```diff
@@ -181,12 +179,6 @@ object ActiveServerList : Listenable {
     internal var serverList = ServerList(mc).apply { loadFile() }
 
     private val serverListPinger = MultiplayerServerListPinger()
-    private val serverPingerThreadPool: ThreadPoolExecutor = ScheduledThreadPoolExecutor(
-        10,
-        ThreadFactoryBuilder().setNameFormat("Server Pinger #%d")
-            .setDaemon(true)
-            .build()
-    )
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/protocol/rest/v1/game/ServerListFunctions.kt:189]: I should not have approved this PR. There is a reason why we use a scheduled thread pool, not ... spawn (now virtual) threads.

Comment (MukjepScarlet) [reply src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/protocol/rest/v1/game/ServerListFunctions.kt:189]: We can use coroutines or Util.getDownloadWorkerExecutor()`

Comment (1zun4) [reply src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/protocol/rest/v1/game/ServerListFunctions.kt:189]: or we just keep it as intended.

Comment (1zun4) [reply src/main/kotlin/net/ccbluex/liquidbounce/integration/interop/protocol/rest/v1/game/ServerListFunctions.kt:189]: it broke my whole server list

**Pull request merged**